src/kirby/components/toast/config/toast-config.ts

```diff
@@ -0,0 +1,7 @@
+import { ThemeColor } from './../../../helpers/theme-color.type';
+
+export interface ToastConfig {
+  message?: string;
```

Comment:
Why is it optional? When do you wan't to show a toast without a message?
```suggestion
  message: string;
```